frontend/packages/console-app/src/components/quick-starts/utils/explore-serverless-qs.ts

```diff
@@ -0,0 +1,92 @@
+export const exploreServerlessQS = {
+  apiVersion: 'console.openshift.io/v1',
+  kind: 'QuickStarts',
+  metadata: {
+    name: 'explore-serverless',
+  },
+  spec: {
+    version: 2.6,
+    displayName: `Explore Serverless`,
+    duration: 10,
+    description: `Install the Serverless Operator to enable containers, microservices and functions to run "serverless"`,
+    prerequisites: `Release requirements if any Install X number of resources`,
+    introduction: `The Red Hat® OpenShift® Serverless provides an enterprise-grade serverless platform. Based on the open source Knative project, it provides a solution for efficient application development supporting dynamic scale and providing a reliable and secure event-driven mechanism to connect on-cluster and off-cluster event producers, bringing portability and consistency across hybrid and multi-cloud environments.
+        Run anywhereUse Kubernetes and OpenShift to build, scale, and manage serverless applications in any cloud, on-premise, or hybrid environment.
+        Integrate with legacyBuild modern, serverless applications and support legacy apps by using event sources. Manage all of your applications in one place: Red Hat OpenShift.
+        Focus on businessKnative allows app teams to focus on building products. Knative installs on OpenShift using Operators, simplifying installation and automating updates and management.
+        Operations friendlyKnative uses Kubernetes to achieve greater consistency and more granular scalability across applications and teams.
+        In this tour, you will performcomplete three tasks:
+        
```

Comment:
this will not be a part of the introduction markup.
slack links for corresponding conversation-
https://coreos.slack.com/archives/C014NQG3H5L/p1595251675216500
https://coreos.slack.com/archives/C014NQG3H5L/p1595258412218900